### ISSUE TITLE
Minimap is hard to use/Selected targets (ships or planets) hard to find

### ISSUE THREAD

**Issue by fizzfaldt** [endless-sky/endless-sky#4639]
**Is your feature request related to a problem? Please describe.**
I'm very frustrated using the minimap.
For example: I'm always frustrated with the corporate espionage events trying to find the right ship.
Even when it's selected, the minimap is very tiny (I don't think there's a way to zoom the actual map/make it bigger) and the selected ship is barely noticable against the background.  I've had so much trouble I'm not actually sure what it does for the selected ship.. does it blink?  Is it white?
This also is a problem when I'm asking for help from a ship and I'm trying to make it faster by going to that ship.  If I can't see it on the minimap I don't know where to go.

**Describe the solution you'd like**
Maybe the minimap could have a 'target' painted around the currently selected item.
Perhaps we could make the minimap bigger (my resolution is 2560x1600 if that's relevant).
Perhaps we could have a UI element on the minimap (or in the main area) pointing the direction to your selected target.
Perhaps we could add a button to 'auto travel ship towards selected target' and/or 'orient ship towards selected target'
Maybe we could provide an element that describes how far away the target is (either in distance or estimated time to reach ship)

**Describe alternatives you've considered**
EV Nova used to have the Auto-orient button "A" on windows that oriented your ship towards a planet/ship that you were targetting.

**Comment by Amazinite** [comment]
>Perhaps we could make the minimap bigger (my resolution is 2560x1600 if that's relevant).

Sounds like you just need the high DPI plugin. https://github.com/endless-sky/endless-sky-high-dpi

**Comment by beccabunny** [comment]
You can also see the direction of the ship you are targeting from the arrow around the selected ship's icon, which points its position relative to you

**Comment by fizzfaldt** [comment]
I already have high DPI installed and the zoom setting over 100.  The minimap never gets bigger.
![myminimap](https://user-images.githubusercontent.com/2748321/68092936-4d06ca80-fe45-11e9-80ba-bac833736f0a.jpg)
![highdpi-installed](https://user-images.githubusercontent.com/2748321/68092938-4d06ca80-fe45-11e9-9caf-9ef337c9396c.jpg)
![settings](https://user-images.githubusercontent.com/2748321/68092939-4d06ca80-fe45-11e9-9c34-3a77dd88279d.jpg)

**Comment by tehhowch** [comment]
> You can also see the direction of the ship you are targeting from the arrow around the selected ship's icon, which points its position relative to you

Reference image:
![image](https://user-images.githubusercontent.com/20871346/68092954-49c80a80-fe56-11e9-807f-9db5b21fc03a.png)
Note the pip around the target reticle near 10 o'clock

**Comment by fizzfaldt** [comment]
> Note the pip around the target reticle near 10 o'clock
I never noticed that pip.. that helps.

**Comment by Amazinite** [comment]
>I already have high DPI installed and the zoom setting over 100. The minimap never gets bigger.

It looks like the minimap *is* getting bigger. The minimap is typically 300x300, but in that image it's roughly 525x525, which would be expected at a 175 main zoom factor. Perhaps just try to use a larger zoom factor? I believe it should be able to go up to 200.

**Comment by fizzfaldt** [comment]
> It looks like the minimap is getting bigger. The minimap is typically 300x300, but in that image it's roughly 525x525, which would be expected at a 175 main zoom factor. Perhaps just try to use a larger zoom factor? I believe it should be able to go up to 200.

Yes, going to 200 helped.. the minimap is just barely large enough for me at zoom level 200.  Everything is too big.. but it's not that big a deal.  I guess it would be nice if there were multiple controls but I can live with zoom 200.  Thanks!

**Comment by Amazinite** [comment]
>Maybe we could provide an element that describes how far away the target is (either in distance or estimated time to reach ship)

Install a tactical scanner. It provides you the range to your target, among other information.

**Comment by TomGoodIdea** [comment]
Is there anything else we don't have yet, but we want to implement?

**Comment by TheGiraffe3** [comment]
It might be nice to have a radar display size setting, but I'm not sure how hard that would be to implement.

**Comment by TomGoodIdea** [comment]
The radar is already customizable:
https://github.com/endless-sky/endless-sky/blob/00230ea3743189714d9353afc710609d2dfa10c6/data/_ui/interfaces.txt#L1076-L1082

**Comment by TomGoodIdea** [comment]
Alright, I'm closing this as completed.
